packages/types-dev/objects.d.ts

```diff
@@ -635,9 +635,11 @@ declare global {
             /** How the adapter will mainly receive its data. Set this together with @see connectionType */
             dataSource?: 'poll' | 'push' | 'assumption';
             /** A record of ioBroker adapters (including "js-controller") and version ranges which are required for this adapter on the same host. */
-            dependencies?: Depdendencies;
+            dependencies?: Dependencies;
             /** A record of ioBroker adapters (including "js-controller") and version ranges which are required for this adapter in the whole system. */
-            globalDependencies?: Depdendencies;
+            globalDependencies?: Dependencies;
+            /** Similar to `dependencies`, but only checked if the specified adapter is already installed. If the adapter is not installed, the version check will pass */
+            ifInstalledDependencies?: { [adapterName: string]: string }[];
```

Comment:
If we can define a new prop clear do we jsut want to make it a record<string, string> without the array, because every adapter name is unique so the array structure is not needed and jsut adds more "complexity" in handling this code wise.